.github/workflows/sfl-pr-review.md

```
@@ -0,0 +1,185 @@
+  create-pull-request-review-comment:
+    side: RIGHT
+    max: 20
+  submit-pull-request-review:
+    allowed-events: [APPROVE, REQUEST_CHANGES]
+    supersede-older-reviews: true
+    footer: always
+  create-check-run:
+    max: 1
+    name: "SFL Reviewer Approval"
+---
+# Deployed from: HemSoft/set-it-free-loop/deployment/workflows/sfl-pr-review.md@78483bbf7edf0a4f8d3bf2f68e58678da36044ae
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)